lib/ldap.c

```diff
@@ -259,6 +259,13 @@ static int ldap_win_bind(struct connectdata *conn, LDAP *server,
 }
 #endif /* #if defined(USE_WIN32_LDAP) */
 
+#if defined(USE_WIN32_LDAP)
+#define FREE_ON_WINLDAP(x) curlx_unicodefree(x)
```

Comment:
Shouldn't the curlx_unicodefree((c)) into two brackets like it is in other places

Reply:
("brackets" are usually these: `[]` - I don't think you meant those)

I don't understand. Why do we need more parentheses?

Reply:
I mean these () brackets. They are being used in other macros in the code as well. See Curl_safefree()